Updates/Instances/565_gruul.sql

```diff
@@ -141,13 +108,60 @@ INSERT INTO `gameobject` (`guid`, `id`, `map`, `spawnMask`, `position_x`, `posit
 -- SPAWN GROUPS
 -- ============
 
--- INSERT INTO `spawn_group` (`Id`, `Name`, `Type`, `MaxCount`, `WorldState`, `Flags`) VALUES
--- INSERT INTO `spawn_group_entry` (`Id`, `Entry`, `MinCount`, `MaxCount`, `Chance`) VALUES
--- INSERT INTO `spawn_group_spawn` (`Id`, `Guid`, `SlotId`, `Chance`) VALUES
+INSERT INTO `spawn_group` (`Id`, `Name`, `Type`, `MaxCount`, `WorldState`, `Flags`) VALUES
+(@SGGUID+1, 'Gruuls\'Lair - High King Maulgar | Group 1', 0, 0, 0, 7),
+(@SGGUID+2, 'Gruuls\'Lair - Gruul | Group 1', 0, 0, 0, 0),
+(@SGGUID+3, 'Gruuls\'Lair - Lair Brute | Gronn-Priest | Patrol', 0, 0, @SGGUID+1, 0),
```

Comment:
Patrols are always Brutes afair

Reply:
I just took what we had and put it in to spawn_groups, so yes its possible, wll try to check it on wotlk classic before merging this

Reply:
Yeah can confirm, they are always Lair Brutes, updated it now